extension/package.json

```diff
@@ -10,9 +10,10 @@
         "Uchida Taishi"
     ],
     "version": "0.7",
-    "fullName": "MemChaser",
+    "title": "MemChaser",
     "id": "memchaser@quality.mozilla.org",
     "description": "Chasing garbage collection and memory usage",
+    "main": "lib/main.js",
```

Comment:
Ah, that information was missing for me. It makes sense now. I would suggest we take the default and rename the main.js to index.js. Otherwise those changes look fine to me.